core/templates/dev/head/components/AnswerGroupEditorDirective.js

```diff
@@ -45,7 +45,7 @@ oppia.directive('answerGroupEditor', [
           $scope.rulesMemento = null;
           $scope.activeRuleIndex = ResponsesService.getActiveRuleIndex();
           $scope.editAnswerGroupForm = {};
-
+         
```

Comment:
Please proofread your PR (using the "Files Changed" tab) before requesting a review, and remove any changes (like this one) that should not be in there.